public/v4/apps/chatwoot-botpress-bridge.yml

```diff
@@ -0,0 +1,46 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: douglara/chatwoot-botpress-bridge:$$cap_version
+        restart: always
+        environment:
+            RAILS_ENV: production
+            CHATWOOT_ENDPOINT: $$cap_chatwoot_endpoint
+            CHATWOOT_BOT_TOKEN: $$cap_chatwoot_bot_token
+            BOTPRESS_ENDPOINT: $$cap_botpress_endpoint
+            BOTPRESS_BOT_ID: $$cap_botpress_bot_id
+            SECRET_KEY_BASE: $$cap_secret_key_base
+        caproverExtra:
+            containerHttpPort: '3000'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_version
+          label: Chatwoot Botpress Bridge Version
+          defaultValue: '0'
+          description: Check out their docker page for the valid tags https://hub.docker.com/r/douglara/chatwoot-botpress-bridge/tags
+        - id: $$cap_secret_key_base
+          label: Secret Key Base
+          description: The randomized string which is used to verify the integrity of signed cookies. Use `openssl rand -hex 64` to generate a key
+        - id: $$cap_chatwoot_endpoint
+          label: Chatwoot endpoint
+          description: 'Chawoot url example: https://app.chatwoot.com'
+        - id: $$cap_chatwoot_bot_token
+          label: Chatwoot Bot Token
+          description: 'To generate token follow instructions: https://github.com/douglara/chatwoot-botpress-bridge'
+        - id: $$cap_botpress_endpoint
+          label: Botpress endpoint
+          description: 'Botpress url example: https://app.botpress.com'
+        - id: $$cap_botpress_bot_id
+          label: Botpress Bot ID
+          description: 'ID for bot you have created in botpress'
+    instructions:
+        start: >-
+            Bridge to connect chatwoot and botpress.
+        end: >-
+            Aaaand you're Bridge is now successfully deployed! 😄
+
+            Get all instructions to use in https://github.com/douglara/chatwoot-botpress-bridge
+    displayName: 'Chatwoot Botpress Bridge'
+    isOfficial: true
```

Comment:
Please change to false as this image is not maintained by neither Chatwoot or Botpress.